.github/drafts/16521/16521-Code-Scanner-Outline.md

```diff
@@ -0,0 +1,73 @@
+# Introduction
+
+ * Code Scanner Intro
+
+  * Video
+
+# Add to Canvas
+
+Elaborate on the steps to add the widget to canvas.
+
+# Usage
+
+Explain the functionality in brief and provide some use cases where this widget can be helpful to the user.
+
+* **Warehouse Example**- With the help of a code scanner widget, employees can easily keep track of their warehouse inventories. They can use a code scanner in their app to scan codes rather than manually entering them.
+* **Restaurant Example**- Exaplain use of QR codes in Restaurants
+
+# Properties 
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+| Property        | Type                 | Description                                                                                                         | Code Snippet                |
+|-----------------|----------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| Text            | Formatting           | Sets the label text of the widget.                                                                                  | -                           |
+| Visible         | Binding & Formatting | Controls widget's visibility on the page. When turned off, the widget will not be visible when the app is published | {{CodeScanner1.isVisible}}  |
+| Disabled        | Binding & Formatting | Disables input to the widget. The widget will remain visible to the user but a user input will not be allowed.      | {{CodeScanner1.isDisabled}} |
+| Animate Loading | Formatting           | Controls the loading of the widget.                                                                                 | -                           |
+| Tooltip         | Formatting           | It sets a tooltip for the widget. You can add hints or extra information about the required input from the user.    | -                           |
+| value           | Binding              | Fetches the value of scanned code                                                                                   | {{CodeScanner1.value}}      |
+|                 |                      |                                                                                                                     |                             |
+
+
+**//cross-link common properties** 
+
+### Text
+
+### Visible
+
+### Disabled
+
+ ### Animate Loading
```

Comment:
No need to explain the property here. Just adding a cross-reference to Animate loading property description in properties table above would suffice.